src/sentry/integrations/issues.py

```diff
@@ -59,8 +59,7 @@ def get_create_issue_config(self, group, **kwargs):
         """
         event = group.get_latest_event()
         if event is not None:
-            if not options.get("eventstore.use-nodestore"):
-                event.bind_node_data()
+            event.bind_node_data()
```

Comment:
This was an error earlier, since `group.get_latest_event()` uses `eventstore.get_events()` rather than `eventstore.get_event_by_id()` this shouldn't be behind the option